pkg/ensurance/cm/advanced_cpu_manager.go

```diff
@@ -295,6 +299,20 @@ func (m *AdvancedCpuManager) getSharedCpu() cpuset.CPUSet {
 	return sharedCPUSet
 }
 
+func (m *AdvancedCpuManager) GetExclusiveCpu() cpuset.CPUSet {
+	exclusiveCPUSet := cpuset.NewCPUSet()
+	for _, pod := range m.activepods() {
+		for _, container := range pod.Spec.Containers {
+			if cset, ok := m.state.GetCPUSet(string(pod.UID), container.Name); ok {
```

Comment:
Can the exclusiveCPUSet is a value maintained in cpuset manager, not computed when it be got?